plugins/azure-devops/src/routes.ts

```diff
@@ -13,8 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import { createRouteRef } from '@backstage/core-plugin-api';
 
-export const azureDevOpsRouteRef = createRouteRef({
-  title: 'azure-devops',
+export const azurePipelinesEntityContentRouteRef = createRouteRef({
+  title: 'Azure Pipelines Entity Content',
```

Comment:
Just a small nit here, when this lands https://github.com/backstage/backstage/pull/7702, `title` is not longer set by the `routeRef` and `id` becomes mandatory. Is this something we could incorporate into this change too see as your playing around with them anyway?

Reply:
For sure, I've made this change, thanks

Reply:
@marleypowell you might want to do this as well

Reply:
> @marleypowell you might want to do this as well

Thanks I will make sure to include it in my PR.